web/models/project.ex

```diff
@@ -22,9 +22,11 @@ defmodule CodeCorps.Project do
     belongs_to :organization, CodeCorps.Organization
     has_one :stripe_connect_plan, CodeCorps.StripeConnectPlan
 
+
     has_many :donation_goals, CodeCorps.DonationGoal
     has_many :project_categories, CodeCorps.ProjectCategory
     has_many :project_skills, CodeCorps.ProjectSkill
+    has_many :stripe_connect_subscriptions, through: [:stripe_connect_plan, :stripe_connect_subscriptions]
```

Comment:
This can eventually not be a `through` if we make it so subscriptions belong to project directly, instead of plan